src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -267,7 +275,12 @@ def get_counter(self, sub_id, table_name):
         if not oid:
             return
 
-        if oid in self.oid_lag_name_map:
+        if oid in self.mgmt_oid_name_map:
+            # TODO: mgmt counters not available through SNMP right now
+            # COUNTERS DB does not have suppot for generic linux (mgmt) interface counters
+            mibs.logger.warning('management interface counters not available through SNMP right now')
```

Comment:
>warning [](start = 24, length = 7)

Is it too verbose in syslog?

Reply:
This gets printed in syslog when someone tries to querry for mgmt interface counters through SNMP. I think it is good to print at least in syslog that we return 0 because counters are not supported through SNMP for mgmt interface